scripts/src/java/org/oppia/android/scripts/coverage/CoverageRunner.kt

```diff
@@ -60,7 +60,9 @@ class CoverageRunner(
     val sfStartIdx = coverageData.indexOfFirst {
       it.startsWith("SF:") && it.substringAfter("SF:").substringAfterLast("/") == extractedFileName
     }
-    if (sfStartIdx == -1) throw IllegalArgumentException("File not found")
+    if (sfStartIdx == -1) throw IllegalArgumentException(
+      "Coverage data not found for the file: $extractedFileName"
+    )
```

Comment:
```suggestion
    if (sfStartIdx == -1) throw IllegalArgumentException(
      "Coverage data not found for the file: $extractedFileName"
    )
```
```suggestion
    require(sfStartIdx != -1) {
      "Coverage data not found for the file: $extractedFileName"
    }
```

More Kotlin-idiomatic way to handle this.

Reply:
Got it! Updated accordingly. Thanks.